modules/aruco/test/test_aruco_utils.hpp

```diff
@@ -0,0 +1,101 @@
+#include "test_precomp.hpp"
+namespace opencv_test {
+    namespace {
+
+        static inline double deg2rad(double deg) { return deg * CV_PI / 180.; }
```

Comment:
avoid indentation in namespaces

Reply:
fixed

Reply:
@alalek thanks for the review)
do you have any more questions?